command/volume_delete.go

```diff
@@ -136,7 +136,31 @@ func (c *VolumeDeleteCommand) deleteCSIVolume(client *api.Client, volID string,
 		}
 	}
 
-	err := client.CSIVolumes().DeleteOpts(&api.CSIVolumeDeleteRequest{
+	// get a CSI volume that matches the given prefix or a list of all matches
+	// if an exact match is not found.
+	stub, possible, err := getByPrefix[api.CSIVolumeListStub]("volumes", client.CSIVolumes().List,
```

Comment:
if the volID is uuid-shaped, could assume it's supposed to be exact and save this api call

Reply:
The ID of a CSI volume isn't UUID-shaped, so we can't actually assume that.

Reply:
Done! (below)